### PR TITLE
Add automatic receipt printing

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

File: lib/trader.js [hunk 1]
```diff
@@ -650,9 +650,11 @@ Trader.prototype.pollHandler = function pollHandler (data) {
 
     if (this.receiptPrintingActive = !!data.receiptInfo) {
       this.smsReceiptActive = data.receiptInfo.sms
+      this.automaticReceiptPrintActive = data.receiptInfo.automaticPrint
       this.receiptOptions = _.omit(['sms'], data.receiptInfo)
     } else {
       this.smsReceiptActive = null
+      this.automaticReceiptPrintActive = null
       this.receiptOptions = null
```
Comment (siiky) [inline lib/trader.js]: This will conflict with #815.

File: ui/src/app.js [hunk 2]
```diff
@@ -123,6 +125,7 @@ function processData (data) {
   if (data.tx && data.tx.discount) setCurrentDiscount(data.tx.discount)
   if (data.receiptStatus) setReceiptPrint(data.receiptStatus, null)
   if (data.smsReceiptStatus) setReceiptPrint(null, data.smsReceiptStatus)
+  if (data.automaticPrint) setAutomaticPrint(data.automaticPrint)
```
Comment (siiky) [inline ui/src/app.js:133]: Shouldn't `setAutomaticPrint()` be run unconditionally here? There's an if inside on `data.automaticPrint` already.

**Review by siiky — APPROVED** [review]
Only one small question, think it looks good.

File: lib/trader.js (hunk 1 above)
Comment (chaotixkilla) [reply lib/trader.js]: Yeah. I'll remind @RafaelTaranto to wait for https://github.com/lamassu/lamassu-machine/pull/815 merge and then I'll rebase this

File: ui/src/app.js (hunk 2 above)
Comment (chaotixkilla) [reply ui/src/app.js:133]: That would make the `else` part be executed everytime, which we do not want

Comment (siiky) [reply ui/src/app.js:133]: Hmm I don't really understand it, but you say it's relevant, so be it.

**Review by siiky — APPROVED** [review]

**Pull request merged**